packages/console/src/pages/ApiResourceDetails/index.module.scss

```diff
@@ -12,8 +12,10 @@
   .info {
     display: flex;
 
-    .imagePlaceholder {
+    .icon {
       margin-left: _.unit(2);
+      width: 60px;
+      height: 60px;
```

Comment:
better use object-fit to center align the img

Reply:
updated